smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/RuntimeClientPlugin.java

```diff
@@ -433,13 +454,16 @@ public Builder resolveFunction(SymbolReference resolveFunction) {
          * {@link #inputConfig} must also be set.
          *
          * @param resolveFunction Function used to convert input to resolved.
-         * @param additionalParameters Additional parameters to be generated as resolve function input.
+         * @param resolveFunctionParamsSupplier Function which returns params to be passed as resolve function input.
```

Comment:
Isn't this function only returns additional parameters other than the previously resolved config?

Reply:
Added in [`015d5e4` (#353)](https://github.com/awslabs/smithy-typescript/pull/353/commits/015d5e4da24d2d3f8892af9f17768db3f70107e3)